server/resource_server/emails/verification/verifier.go

```diff
@@ -0,0 +1,71 @@
+package verification
+
+import (
+	"fmt"
+	"globe-and-citizen/layer8/server/resource_server/emails/sender"
+	"globe-and-citizen/layer8/server/resource_server/emails/verification/code"
+	"globe-and-citizen/layer8/server/resource_server/models"
+	"os"
+	"time"
+)
+
+type EmailVerifier struct {
+	adminEmailAddress string
+
+	emailSenderService sender.Service
+	codeGenerator      code.Generator
+
+	now func() time.Time
+}
+
+func NewEmailVerifier(
+	adminEmailAddress string,
+	emailSenderService sender.Service,
+	codeGenerator code.Generator,
+	now func() time.Time,
+) *EmailVerifier {
+	verifier := new(EmailVerifier)
+
+	verifier.adminEmailAddress = adminEmailAddress
+	verifier.emailSenderService = emailSenderService
+	verifier.codeGenerator = codeGenerator
+	verifier.now = now
+
+	return verifier
+}
+
+func (v *EmailVerifier) GenerateVerificationCode(user *models.User) string {
+	return v.codeGenerator.GenerateCode(user.Email)
+}
+
+func (v *EmailVerifier) SendVerificationEmail(user *models.User, verificationCode string) error {
+	return v.emailSenderService.SendEmail(
+		&models.Email{
+			From:    v.adminEmailAddress,
+			To:      user.Email,
+			Subject: os.Getenv("VERIFICATION_EMAIL_SUBJECT"),
```

Comment:
Can we not just hard code the subject here instead of getting that from a .env? Was this necessary? Although in future we will have proper templates, but getting the things which can are not required to be set in .env can be used directly, what do you think?